libraries/standard/http/utest/CMakeLists.txt

```diff
@@ -22,15 +22,20 @@ list(APPEND mock_define_list
 
 # ================= Create the library under test here (edit) ==================
 
+set(logging_sources 
```

Comment:
Should these be in its own file?

Reply:
@aggarw13 We could add these to a separate file in `tools/cmake`.

Reply:
Yeah, will add them in the `tool/cmake` section

Reply:
Added